docs/source/contributing/branch-policy.md

```diff
@@ -33,14 +33,19 @@ legacy
     When opening a pull request, the contributor must open it against `main`.
     If the pull request is a feature or a bugfix, and if the release manager deems it useful to the latest version's branch, they may ask the contributor to backport it to that branch.
 
+`17.x.x`
+:   `17.x.x` is the current stable and actively developed branch in git.
+    Upon the final release of version `18.0.0`, the `17.x.x` branch line will be no longer supported.
+    Any new feature will be generally released to the `main` branch, and only backported to older versions if the Volto Team approves it.
```

Comment:
@sneridagh The word "released" is confusing here. You mean "merged", right?

Reply:
How about this?

```
    Any new feature will be merged into the `main` branch, and only backported to and released in older versions, if the Volto Team approves it.
```